scripts/qa_crowbarsetup.sh

```diff
@@ -4184,7 +4184,8 @@ function oncontroller_testsetup
 
     wait_for 40 5 "timeout -k 20 10 ssh -o UserKnownHostsFile=/dev/null $ssh_target true" "SSH key to be copied to VM"
 
-    if ! ssh $ssh_target curl $clouddatadns/test ; then
+    [[ $http_proxy ]] && http_proxy_env="http_proxy=$http_proxy" || http_proxy_env=""
+    if ! ssh $ssh_target $http_proxy_env curl -s --fail http://suse.com ; then
```

Comment:
@gonzolino until now I would say it is `$clouddata`, but with https://github.com/SUSE-Cloud/automation/pull/1933 we kill it and then I would propose to run the test with `$reposerver_fqdn`